pkg/remote/codec/thrift/thrift.go

```diff
@@ -242,7 +242,7 @@ func (c thriftCodec) Name() string {
 
 // MessageWriterWithContext write to thrift.TProtocol
 type MessageWriterWithContext interface {
-	Write(ctx context.Context, oprot thrift.TProtocol) error
+	Write(ctx context.Context, method string, isClient bool, oprot thrift.TProtocol) error
```

Comment:
MessageWriterWithMethodWithContext

Reply:
No need to add `isClient` parameter, because we could known whether it's client by args or result, if writing args, it means client, otherwise it's server.